### ISSUE TITLE
Add conditional option to disable marketing pages in the application

### ISSUE THREAD

**Issue by agjohnson** [readthedocs/readthedocs.org#9512]
Opening this conversation up, but this will be a fairly large task, so would like to start talking about options here.

With the new application templates and our new website, marketing content that currently lives in the application will no longer be used after we settle into the new website and especially once the beta instance is the new prod.

Remaining content pages in our applications:

- [x] Sustainability/donate pages
- [x] Security page
- [x] Commercial pricing page `PricingView`
- [x] Commercial enterprise page `EnterpriseView`
- [x] corporate/features.html
- [x] corporate/services.html

Some of the changes to discuss:

* [ ] Conditionally active the views/URLs in the application, based on a setting value or the new application theme is in use?
* [x] Make the sign in page the default `/` page if we are not serving marketing pages in the instance?
* [x] Keep the support page in the application though, I assume
* [ ] Content that needs to move out from the application to the website?
* Any other concerns?

I'd like to break this work up and at least address the default, unauthenticated page that the application serves.

**Comment by ericholscher** [comment]
I'd like to do this work along with removing our branding in general from the code. Copied from an old Trello issue:

This probably relies a little bit on having an unbranded version of our
"Community Version" or something that isolates our brand from our code. We'll lump in changing the org name at the same time, because why not.

- Finally drop or convert the `readthedocs` GitHub user to an org, or remove and rename the organization, or whatever avoids conflicts.
- Change repository name to \`readthedocs-core\` or anything but \`readthedocs.org\`
- Update readme and some other places with thoughts about why
- Change org name
- Kill the logo in the repo
- Update pieces pointing to rtfd/readthedocs.org

  - In ops salt states
  - Readme

**Comment by agjohnson** [comment]
I've been chipping away at this, and guessing we'll continue this sort of pattern for moving content around -- detect when the new dashboard is in use and remove the view or redirect. The main view is conditionally a redirect to the login view on the new dashboard, but no other major pages have moved out.

There are some other pages like the donation pages. We've talked about removing those entirely.

I haven't made any changes to remove branding yet. That work is still going to be a bit extra effort on top of a change like the one here, especially as the new dashboard moves closer to being a required dependency of the application. We would need a third layer to add our branding back in over the application + ext-theme layers. This still feels related, but would be separate.

**Comment by ericholscher** [comment]
Yea, we should probably figure out any other pages we want to remove or redirect. I think the one-time donation page is a prime candidate to just go away, and the gold page can redirect to the new website pricing page.

**Comment by agjohnson** [comment]
Yeah, donation page definitely, and commercial has some pages like pricing/etc too. Those can all start to get cut from the application for the new dashboard very easily. And we can start to redirect to the website for the current prod dashboard at some point too.

The gold page in the user admin has some interactive components, so probably is a legitimate UI. But maybe all of the long form copy on those pages should move to the website -- the ad removal page and gold subscription page have a fair amount of copy on them.

**Comment by ericholscher** [comment]
Yea, the dashboard pages where you're actually changing state definitely need to stay in the application. Guess I was mostly thinking about https://readthedocs.org/sustainability/ redirecting to https://about.readthedocs.com/pricing/#/community

**Comment by agjohnson** [comment]
Ah gotcha, yeah that's the page I'm referring to as well. That's a good one to remove, and yeah I don't think we have too many more.

Looking through URLs, we have this one too: https://readthedocs.org/security/ :upside_down_face:

**Comment by ericholscher** [comment]
Ha, that one should be safe to redirect 😆

**Comment by agjohnson** [comment]
Also, we can split up the gold/donation changes and the marketing page changes, the marketing changes can more immediately be dropped here.

**Comment by humitos** [comment]
I opened two PRs to remove and redirect old URLs:

* https://github.com/readthedocs/readthedocs.org/pull/12246
* https://github.com/readthedocs/readthedocs-corporate/pull/2008

I don't think there are more immediate work to do here and we can probably close this issue.

> I'd like to do this work along with removing our branding in general from the code

I think this is going to be pretty hard to prioritize and we should open a new issue if we really want to do that.
